2.0/db_models/holdings.py

```
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, Integer, BigInteger, String, Text, ForeignKey
+from sqlalchemy.orm import relationship
+from .base import Base
+import config.settings as settings
```

Pylint (3.3.7)

[refactor] Use 'from config import settings' instead

(R0402)